assets/clp_presto/Dockerfile

```
@@ -0,0 +1,77 @@
+
+# RUN apt-get install -y \
+#     tmux \
+#     vim \
+#     libsnappy-dev \
+#     libdouble-conversion-dev \
+#     libre2-dev \
+#     libgflags-dev \
+#     libboost-all-dev;
+
+# Install Task
+RUN cd /usr/local && sh -c "$(curl --location https://taskfile.dev/install.sh)" -- -d;
```

Hadolint (2.12.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)